### PR TITLE
UI: num unique trackers in details hero subtitle

### PR DESCRIPTION
**Pull request by laurengarcia**
**Reviewer:**
@MariagraziaAlastra 

## Description:
Display number of unique trackers in trackers subview hero.

## Steps to test this PR:
Build this branch and visit sites, click thru to trackers details subview and view hero subtitle.

unique trackers

unique trackers zero

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: shared/js/ui/templates/shared/tracker-networks-text.es6.js [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
 
   let trackersText = isMajorNetworksCount ? ' Major Tracker' : ' Tracker'
   trackersText += (trackerNetworksCount === 1) ? ' Network ' : ' Networks '
+  let finalText = trackerNetworksCount + trackersText + trackersBlockedOrFound(site, trackerNetworksCount)
 
-  const finalText = trackerNetworksCount + trackersText + trackersBlockedOrFound(site, trackerNetworksCount)
+  if (includeUniqueTrackersCount && trackerNetworksCount > 0) {
+    const uniqueTrackersText = site.trackersCount === 1 ? ' Unique Tracker In ' : ' Unique Trackers In '
+    finalText = site.trackersCount + uniqueTrackersText + finalText
+    return bel`${finalText}`
```
Comment (andrey-p) [inline shared/js/ui/templates/shared/tracker-networks-text.es6.js]: Tiny nitpick, this `return` is unnecessary.

File: shared/scss/views/_tracker-networks.scss [hunk 2]
```diff
@@ -1,6 +1,12 @@
-.tracker-networks .hero__icon {
-    width: 58px;
-    height: 65px;
+.tracker-networks {
+    .hero__icon {
+        width: 58px;
+        height: 65px;
+    }
+    .hero__subtitle {
+        height: 40px;
```
Comment (andrey-p) [inline shared/scss/views/_tracker-networks.scss]: What do you think about reducing this to 32px? It'll help keep the margins a bit more consistent, and leave less empty space when there's only one line of text.

screenshot from 2018-02-02 12-27-34

**Comment by andrey-p** [comment]
Left a coupla comments. :+1: otherwise. Feel free to self-merge when ready.

File: shared/js/ui/templates/shared/tracker-networks-text.es6.js (hunk 1 above)
Comment (laurengarcia) [reply shared/js/ui/templates/shared/tracker-networks-text.es6.js]: True True

**Comment by laurengarcia** [comment]
Thanks

**Pull request merged**